packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -301,6 +305,14 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
     if (this._autoOpened) {
       this._close(true);
     }
+    this.__connected = false;
+  }
+
+  /** @protected */
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.__connected = true;
```

Comment:
Consider using `_isConnected` like the property used in `vaadin-grid-sorter`.

Reply:
Done